specutils/tests/test_loaders.py

```diff
@@ -1273,3 +1273,64 @@ def test_galah_guess(remote_data_path):
         assert isinstance(spectra[1].uncertainty, StdDevUncertainty)
         assert spectra[1].meta.get("label") is not None
         assert spectra[1].meta.get("header") is not None
+
+
+# We cannot use remote_access directly in the MIRI MRS tests, because
+# the test functions are called once for every file in the remote access
+# list, but we need to feed the entire set to the functions at once. We
+# store the file names in a list and use a dummy test method to load the
+# list via the remote_access machinery.
+#
+# The zenodo record contains the entire set of 12 _x1d files corresponding
+# to this data set. We use just a few files for the current tests. Maybe
+# more files will be needed when more functionality is added later on.
+filename_storage = []
+@remote_access([
+    {'id': '5082863', 'filename': 'combine_dithers_all_exposures_ch1-long_x1d.fits'},
+    {'id': '5082863', 'filename': 'combine_dithers_all_exposures_ch1-medium_x1d.fits'},
+    {'id': '5082863', 'filename': 'combine_dithers_all_exposures_ch1-short_x1d.fits'},
+])
+def test_loaddata_miri_mrs(remote_data_path):
+    filename_storage.append(remote_data_path)
```

Comment:
Indeed. But the `@remote_access` decorator doesn't seem to work with setup functions. Apparently it must be applied to `test_` functions.

Reply:
The "merge from astropy/main" commits were generated by GitHub itself. Maybe this is why I can't see then in my local copy? 
<img width="491" alt="image" src="https://user-images.githubusercontent.com/3577145/125293272-40609800-e2f1-11eb-8cdb-8b1eb53b4bea.png">

Reply:
Now it apparently contains changes from #833 and possibly other unrelated PRs; looks a bit like you might have rebased this branch onto `astropy/main` instead of `git rebase `astropy/main read-many`?

Reply:
Possibly. I'll check on that.